cumulusci/tasks/bulkdata/generate_and_load_data.py

```diff
@@ -0,0 +1,105 @@
+import os
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.tasks.bulkdata import LoadData
+from cumulusci.utils import temporary_dir
+from cumulusci.core.config import TaskConfig
+from cumulusci.core.utils import import_global
+from cumulusci.core.exceptions import TaskOptionsError
+
+
+class GenerateAndLoadData(BaseSalesforceApiTask):
+    """ Orchestrate creating tempfiles, generating data, loading data, cleaning up tempfiles and batching."""
```

Comment:
In the docstring or in the task_docs?

Reply:
Usually for tasks it should go in task_docs, which is the source when you run `cci task doc > docs/tasks.rst` to collate the task docs into the file that Sphinx builds. But I don't think this one will be included there since it's not configured in cumulusci.yml by default.